db/seeds.rb

```diff
@@ -1,7 +1,6 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+# Создадим записи в базе данных для проверки, восклицательный знак после create стоит для вызова исключения в случае ошибки создания записи.
+Card.create!([
+  {original_text:"Привет. Как дела?", translated_text:"Hello. How are you?", review_date: Date.current},
```

Comment:
Line is too long. [104/80]<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.